pkg/cmd/release/create/create.go

```diff
@@ -244,6 +244,22 @@ func createRun(opts *CreateOptions) error {
 				} else {
 					headRef = "HEAD"
 				}
+			} else {
+				// Local tag exists and target not specified so use tag
+				// SHA as target. This prevents the scenario where the
+				// user has a local unpushed tag and then when creating a
+				// release the API creates a new tag with the same name
+				// pointing to the repos default branch. This is confusing
+				// when the local unpushed tag was pointing to a different SHA.
+				// This will ensure that when the remote tag gets created
+				// it points to the correct commit.
+				if opts.Target == "" {
+					tagSha, err := gitTagSha(opts.TagName)
```

Comment:
Minor: this is a 2nd git lookup for the same tag name. Could ` gitTagInfo` be expanded to return both `tagDescription` and `tagSha` in a single call so we don't have to shell out to git twice?

Reply:
Yup good call, was able to work that in.